web/src/components/App/index.js

```diff
@@ -164,8 +186,10 @@ class App extends Component {
             </Navbar.Brand>
           </Navbar.Header>
           <Navbar.Collapse>
-            <Nav pullLeft>
-              <NavItem>({dbName})</NavItem>
+            <Nav pullLeft activeKey={dbInfo.key}>
+              {otherDbs.map(db => {
+                return <NavItem key={db.key} eventKey={db.key} href={db.path}>({db.name})</NavItem>;
```

Comment:
Let's also add key to the navbar.
```suggestion
                return <NavItem key={db.key} eventKey={db.key} href={db.path}>{db.key} ({db.name})</NavItem>;
```